src/hooks/v3/transactor/V3IssueErc20TokenTx.ts

```diff
@@ -0,0 +1,56 @@
+import { t } from '@lingui/macro'
+import { V2V3ContractsContext } from 'contexts/v2v3/V2V3ContractsContext'
+import { useContext } from 'react'
+import invariant from 'tiny-invariant'
+
+import { ProjectMetadataContext } from 'contexts/projectMetadataContext'
+import { TransactionContext } from 'contexts/transactionContext'
+import {
+  handleTransactionException,
+  TransactorInstance,
+} from 'hooks/Transactor'
+
+export function useV3IssueErc20TokenTx(): TransactorInstance<{
+  name: string
+  symbol: string
+}> {
+  const { transactor } = useContext(TransactionContext)
+  const { contracts } = useContext(V2V3ContractsContext)
+  const { projectId, cv } = useContext(ProjectMetadataContext)
+
+  return ({ name, symbol }, txOpts) => {
+    try {
+      invariant(
+        transactor && projectId && contracts?.JBController && name && symbol,
+      )
+      return transactor(
+        contracts.JBTokenStore,
+        'issueFor',
```

Comment:
In V3 we issue tokens using a different method on a different contract. So, this new hook does that :)